src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -142,9 +142,23 @@ private void LogSuccessfulTelemetryToClient(AuthenticationResult authenticationR
                 telemetryEventDetails.SetProperty(TelemetryConstants.PopToken, authenticationResult.TokenType.Equals(Constants.PoPTokenType));
                 telemetryEventDetails.SetProperty(TelemetryConstants.RemainingLifetime, (authenticationResult.ExpiresOn - DateTime.Now).TotalMilliseconds);
                 telemetryEventDetails.SetProperty(TelemetryConstants.ActivityId, authenticationResult.CorrelationId);
+                telemetryEventDetails.SetProperty(TelemetryConstants.RefreshOn, 
```

Comment:
Why are we logging this? What's reporting value of datetime when refresh has to happen? Would something like time remaining time until refresh (kind of like remaining lifetime until expiration above) or refresh on duration be more useful? I also think this value is always non-null.

Reply:
It was requested by the folks. Maybe @rymeskar remembers what they plan to do with `RefreshOn` ?